### PR TITLE
Menu

### PR DESCRIPTION
**Pull request by farouqdaif1**
Add Dynamic menu 
Add linters for js

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,19 +29,28 @@
...
-              <li>About</li>
-              <li>Contact</li>
+            <div id="close">      
+              <button type="button" class="close-btn">
+              <img src="image/Enabled.png" alt="x"/>
+              </button>
+              </div>
+          <div >
+            <ul class="ul2">
+              <li><a href="#work">Portoflio</a></li>
```
Comment (Davidosky007) [inline index.html:46]: * Kindly Fix the desktop nav links as the mobile menu options are visible on desktop
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @adamteddychang 👋, 

Good job so far! 💪 
* Descriptive commit messages ✔️ 
* No linter errors ✔️ 
* Profesional looking readme file 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -29,19 +29,28 @@
...
-              <li>Contact</li>
+            <div id="close">      
+              <button type="button" class="close-btn">
+              <img src="image/Enabled.png" alt="x"/>
+              </button>
+              </div>
+          <div class="toul">
+            <ul class="ul2" onclick="showMe()">
+              <li><a href="#work">Portoflio</a></li>
+              <li><a href="#about">About</a></li>
```
Comment (carlos-ssh) [inline index.html:47]: - Follow the requested design guidelines:
Screen Shot 2021-10-04 at 17 44 18
#
_Yours:_
Screen Shot 2021-10-04 at 17 45 18

File: index.html [hunk 3]
```diff
@@ -29,19 +29,28 @@
...
+              <li><a href="#contact">Contact</a></li>
               <li><i class="fas fa-envelope-square"></i></li> 
             </ul>
           </div>
-
+        </div>
+
```
Comment (carlos-ssh) [inline index.html:53]: - Provide a License in your README file.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @farouqdaif1 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Farouq Daif, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**